src/entities/Publication.ts

```diff
@@ -35,7 +35,7 @@ export class Publication {
   rssURL: string;
 
   @Field()
-  @Property({ unique: true })
+  @Property() // Couldn't create multiple users with unique constraint active
```

Comment:
I would suggest removing this comment and adding this information to the "Changes Made" in the README. The comment doesn't make sense in the context of the code, it only makes sense when someone is comparing the previous version of this line of code, to the current. Therefore, it doesn't belong in code, rather version control.

Reply:
Done